src/main/java/com/hcl/appscan/issuegateway/appscanprovider/asoc/ASOCCommentHandler.java

```diff
@@ -71,5 +79,6 @@ public void submitComments(PushJobData jobData, List<String> errors, Map<String,
 
 	private static class Comment {
 		public String Comment;
+		public Comment[] Items;

```

Comment:
Why is this array added here ??

Reply:
This change is part of Vishal's commit. @vishalhcl-5960  Please could you check and confirm the review comment by Tanmoy.

Reply:
@tanmoyguha-hcl Actually there is difference in the response body of the GET "/api/v4/Issues/ISSUEID/Comments" with respect to the v2 version of same API. The response of the v2 API directly contains a required array while in the response of the v4 API there is parent JSON under which we get the required "Items" array.